iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/planner/OperatorTreeGenerator.java

```diff
@@ -2451,6 +2457,23 @@ public Operator visitLastQueryCollect(
     return new LastQueryCollectOperator(operatorContext, children);
   }
 
+  @Override
+  public Operator visitLastQueryTransform(
+      LastQueryTransformNode node, LocalExecutionPlanContext context) {
+    List<Operator> children = dealWithConsumeChildrenOneByOneNode(node, context);
```

Comment:
No need to call this if there is only one child. Just call like the following:

`node.getChild().accept(this, context);`